tests/testthat/test-lancelot-support.R

```diff
@@ -533,6 +536,14 @@ test_that("lancelot_compare combines likelihood correctly", {
     deaths_carehomes_inc = 2:7,
     deaths_comm_inc = 1:6,
     deaths_hosp_inc = 3:8,
+    deaths_hosp_inc_0_49 = 3:8,
+    deaths_hosp_inc_50_54 = 3:8,
+    deaths_hosp_inc_55_59 = 3:8,
+    deaths_hosp_inc_60_64 = 3:8,
+    deaths_hosp_inc_65_69 = 3:8,
+    deaths_hosp_inc_70_74 = 3:8,
+    deaths_hosp_inc_75_79 = 3:8,
+    deaths_hosp_inc_80_plus = 3:8,
```

Comment:
might want to have different values for each of the above

Reply:
Done